src/components/editor/ResourceTemplate.jsx

```diff
@@ -7,7 +7,7 @@ import ResourceTemplateForm from './ResourceTemplateForm'
 import { rootResourceTemplateLoaded } from 'actions/index'
 import { getResourceTemplate } from 'sinopiaServer'
 import { rootResource } from 'selectors/resourceSelectors'
-import ResourceURIMessage from './resourceURIMessage'
+import ResourceURIMessage from './ResourceURIMessage'
```

Comment:
wow - this worked with the wrong case?  my world is broken.

Reply:
ah, no, i renamed the file too (that was my first change).

actually, i think it did work without the case change, but i did it the other way (renamed the file, ran the tests successfully iirc, got called out by eslint on the import when circle ran the build).